File: benchmark/results/java-invocation-script/run-001/b-baseline/output/com/metaengine/demo/identity/services/UserService.java
package com.metaengine.demo.identity.services;

import com.metaengine.demo.identity.aggregates.User;
import java.util.List;

/** UserService for the identity domain. */
public class UserService {

    /** Create a new User. */
    public User create(User input) {
        throw new UnsupportedOperationException("not implemented");
    }

    /** Find a User by id; may return null. */
    public User findById(String id) {
        throw new UnsupportedOperationException("not implemented");
    }

    /** List users up to the given limit. */
    public List<User> list(int limit) {
        throw new UnsupportedOperationException("not implemented");
    }

    /** Delete a user by id. */
    public void delete(String id) {
        throw new UnsupportedOperationException("not implemented");
    }
}

File: benchmark/results/python-invocation-script/run-005/b-baseline/output/metaengine_demo/identity/services/user_service.py
"""UserService for the identity domain."""
from typing import Optional

from metaengine_demo.identity.aggregates.user import User


class UserService:
    """UserService."""

    def create(self, input: Optional[User]) -> User:
        raise NotImplementedError("not implemented")

    def find_by_id(self, id: str) -> Optional[User]:
        raise NotImplementedError("not implemented")

    def list(self, limit: float) -> list[User]:
        raise NotImplementedError("not implemented")

    def delete(self, id: str) -> None:
        raise NotImplementedError("not implemented")
